WcaOnRails/app/models/locale.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+class Locale < SimpleDelegator
+  include ActionView::Helpers::TextHelper
+  SOME_CHARS = '[\s\S]*?'
```

Comment:
As you will see, I changed it back to "[\s\S]*?" without the use of the regexp multiline mode.
I ran again into perf issues, I'm not sure if the regexp or the multiline caused them, but using the original expression works fast, so I must admit I not interested in investigating why and what caused the issues :p